lib/src/realm_class.dart

```diff
@@ -253,13 +258,22 @@ class Realm {
   /// and [Predicate Programming Guide.](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Predicates/AdditionalChapters/Introduction.html#//apple_ref/doc/uid/TP40001789)
   RealmResults<T> query<T extends RealmObject>(String query, [List<Object> args = const []]) {
     RealmMetadata metadata = _getMetadata(T);
-    final handle = realmCore.queryClass(this, metadata.class_.key, query, args);
-    return RealmResultsInternal.create<T>(handle, this);
+    final queryHandle = realmCore.queryClass(this, metadata.class_.key, query, args);
+    return RealmResultsInternal.create<T>(queryHandle, this);
   }
 
   /// Deletes all [RealmObject]s of type `T` in the `Realm`
   void deleteAll<T extends RealmObject>() => deleteMany(all<T>());
 
+  SubscriptionSet? _subscriptions;
+  SubscriptionSet? get subscriptions {
```

Comment:
Not a fan of this being nullable - I'd rather have it non-nullable and throw an exception if it's used on the wrong realm.